pkgs/top-level/all-packages.nix

```diff
@@ -3986,6 +3986,10 @@ with pkgs;
 
   hebcal = callPackage ../tools/misc/hebcal {};
 
+  hebbot = callPackage ../servers/matrix-hebbot {
+     inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
```suggestion
   inherit (darwin.apple_sdk.frameworks) Security;
```

Reply:
I think splitting the difference here should be correct, right? One space less?